pkg/registry/servicecatalog/instance/strategy.go

```diff
@@ -122,14 +123,24 @@ func (instanceRESTStrategy) PrepareForUpdate(ctx genericapirequest.Context, new,
 		glog.Fatal("received a non-instance object to update from")
 	}
 
+	// Do not allow any updates to the Status field while updating the Spec
+	newServiceInstance.Status = oldServiceInstance.Status
```

Comment:
was this added due to this PR or was it just an oversight that it wasn't here before?

Reply:
This was moved a few lines up from where it was before: no functionality was changed. It was moved originally so that the line preventing the spec change immediately preceded the commented-out lines that were unnecessary because any spec changes are discarded. Since that code is no longer commented-out, then this code movement is not as important any more, but it is still not harmful.

Reply:
ah I missed that it was moved - sorry